Building `konsole` fails (https://github.com/msys2/MINGW-packages/actions/runs/24755582762/job/72427973105?pr=29106#step:9:5610):
```
  [210/235] Linking CXX shared library bin\libkonsoleapp.dll
  FAILED: [code=1] bin/libkonsoleapp.dll lib/libkonsoleapp.dll.a 
  C:\Windows\system32\cmd.exe /C "cd . && D:\M\msys64\ucrt64\bin\g++.exe -march=nocona -msahf -mtune=generic -O2 -pipe -Wp,-D_FORTIFY_SOURCE=2 -fstack-protector-strong -Wp,-D__USE_MINGW_ANSI_STDIO=1 -fno-operator-names -fno-exceptions -Wall -Wextra -Wcast-align -Wchar-subscripts -Wformat-security -Wno-long-long -Wpointer-arith -Wundef -Wnon-virtual-dtor -Woverloaded-virtual -Werror=return-type -Werror=init-self -Werror=undef -Wvla -Wdate-time -Wsuggest-override -Wlogical-op -pedantic -Wzero-as-null-pointer-constant -Wmissing-include-dirs -fdiagnostics-color=always -O3 -DNDEBUG  -shared -Wl,--no-undefined -Wl,--fatal-warnings  -Wl,--export-all-symbols -o bin\libkonsoleapp.dll -Wl,--out-implib,lib\libkonsoleapp.dll.a -Wl,--major-image-version,26,--minor-image-version,4 src/CMakeFiles/konsoleapp.dir/konsoleapp_autogen/mocs_compilation.cpp.obj src/CMakeFiles/konsoleapp.dir/Application.cpp.obj src/CMakeFiles/konsoleapp.dir/MainWindow.cpp.obj src/CMakeFiles/konsoleapp.dir/settings/ConfigurationDialog.cpp.obj src/CMakeFiles/konsoleapp.dir/settings/ConfigDialogButtonGroupManager.cpp.obj src/CMakeFiles/konsoleapp.dir/settings/TemporaryFilesSettings.cpp.obj src/CMakeFiles/konsoleapp.dir/settings/GeneralSettings.cpp.obj src/CMakeFiles/konsoleapp.dir/settings/MemorySettings.cpp.obj src/CMakeFiles/konsoleapp.dir/settings/ProfileSettings.cpp.obj src/CMakeFiles/konsoleapp.dir/settings/TabBarSettings.cpp.obj src/CMakeFiles/konsoleapp.dir/settings/ThumbnailsSettings.cpp.obj src/CMakeFiles/konsoleapp.dir/pluginsystem/IKonsolePlugin.cpp.obj src/CMakeFiles/konsoleapp.dir/pluginsystem/PluginManager.cpp.obj src/CMakeFiles/konsoleapp.dir/delegates/ProfileShortcutDelegate.cpp.obj  lib/libkonsoleprivate.dll.a  D:/M/msys64/ucrt64/lib/libKF6NotifyConfig.dll.a  lib/libkonsoleprivate_core.a  lib/libkonsolecolorscheme.a  lib/libkeyboardtranslator.a  D:/M/msys64/ucrt64/lib/libKF6XmlGui.dll.a  D:/M/msys64/ucrt64/lib/libKF6ConfigWidgets.dll.a  lib/libkonsolehelpers.a  lib/libkonsolecontainers.a  D:/M/msys64/ucrt64/lib/libKF6NewStuffWidgets.dll.a  D:/M/msys64/ucrt64/lib/libKF6NewStuffCore.dll.a  D:/M/msys64/ucrt64/lib/libKF6Attica.dll.a  D:/M/msys64/ucrt64/lib/libQt6PrintSupport.dll.a  -lcomdlg32  -lwinspool  D:/M/msys64/ucrt64/lib/libKF6TextWidgets.dll.a  D:/M/msys64/ucrt64/lib/libKF6SonnetUi.dll.a  D:/M/msys64/ucrt64/lib/libz.dll.a  D:/M/msys64/ucrt64/lib/libicuuc.dll.a  D:/M/msys64/ucrt64/lib/libicuin.dll.a  D:/M/msys64/ucrt64/lib/libKF6Bookmarks.dll.a  D:/M/msys64/ucrt64/lib/libKF6KIOWidgets.dll.a  D:/M/msys64/ucrt64/lib/libKF6KIOGui.dll.a  D:/M/msys64/ucrt64/lib/libKF6WindowSystem.dll.a  D:/M/msys64/ucrt64/lib/libKF6KIOCore.dll.a  D:/M/msys64/ucrt64/lib/libKF6Crash.dll.a  D:/M/msys64/ucrt64/lib/libKF6JobWidgets.dll.a  D:/M/msys64/ucrt64/lib/libKF6Notifications.dll.a  D:/M/msys64/ucrt64/lib/libKF6Service.dll.a  D:/M/msys64/ucrt64/lib/libQt6Xml.dll.a  D:/M/msys64/ucrt64/lib/libKF6CoreAddons.dll.a  D:/M/msys64/ucrt64/lib/libKF6Completion.dll.a  D:/M/msys64/ucrt64/lib/libKF6IconThemes.dll.a  D:/M/msys64/ucrt64/lib/libKF6ColorScheme.dll.a  D:/M/msys64/ucrt64/lib/libKF6I18n.dll.a  D:/M/msys64/ucrt64/lib/libKF6GuiAddons.dll.a  D:/M/msys64/ucrt64/lib/libKF6WidgetsAddons.dll.a  D:/M/msys64/ucrt64/lib/libKF6ConfigGui.dll.a  D:/M/msys64/ucrt64/lib/libKF6ConfigCore.dll.a  D:/M/msys64/ucrt64/lib/libQt6Multimedia.dll.a  D:/M/msys64/ucrt64/lib/libQt6Network.dll.a  -lws2_32  D:/M/msys64/ucrt64/lib/libQt6Widgets.dll.a  D:/M/msys64/ucrt64/lib/libQt6Gui.dll.a  -ld3d11  -ldxgi  -ldxguid  -ld3d12  D:/M/msys64/ucrt64/lib/libQt6Core.dll.a  -lmpr  -luserenv  -lkernel32 -luser32 -lgdi32 -lwinspool -lshell32 -lole32 -loleaut32 -luuid -lcomdlg32 -ladvapi32 && cd ."
  D:/M/msys64/ucrt64/lib/gcc/x86_64-w64-mingw32/15.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: lib/libkonsolecontainers.a(ContainerList.cpp.obj):ContainerList.cpp:(.text+0x20): multiple definition of `Konsole::ContainerList::containerSelected(Konsole::ContainerInfo const&)'; lib/libkonsoleprivate.dll.a(libkonsoleprivate_dll_d000906.o):(.text+0x0): first defined here
  D:/M/msys64/ucrt64/lib/gcc/x86_64-w64-mingw32/15.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: lib/libkonsolecontainers.a(ContainerList.cpp.obj):ContainerList.cpp:(.text+0x620): multiple definition of `Konsole::ContainerList::addContainerSections(QMenu*)'; lib/libkonsoleprivate.dll.a(libkonsoleprivate_dll_d000909.o):(.text+0x0): first defined here
  D:/M/msys64/ucrt64/lib/gcc/x86_64-w64-mingw32/15.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: lib/libkonsolecontainers.a(ContainerList.cpp.obj):ContainerList.cpp:(.text+0xce0): multiple definition of `Konsole::ContainerList::refreshContainers()'; lib/libkonsoleprivate.dll.a(libkonsoleprivate_dll_d000907.o):(.text+0x0): first defined here
  D:/M/msys64/ucrt64/lib/gcc/x86_64-w64-mingw32/15.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: lib/libkonsolecontainers.a(ContainerList.cpp.obj):ContainerList.cpp:(.text+0x12d0): multiple definition of `Konsole::ContainerList::ContainerList(QObject*)'; lib/libkonsoleprivate.dll.a(libkonsoleprivate_dll_d000911.o):(.text+0x0): first defined here
  collect2.exe: error: ld returned 1 exit status
  ninja: build stopped: subcommand failed.
  ==> ERROR: A failure occurred in build()
```

So let's try without it.

--- a/mingw-w64-lokalize/PKGBUILD
+++ b/mingw-w64-lokalize/PKGBUILD
@@ -4,7 +4,7 @@ source "$(dirname ${BASH_SOURCE[0]})"/../mingw-w64-PKGBUILD-common/kde-framework
 _realname=lokalize
 pkgbase="mingw-w64-${_realname}"
 pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}")
-pkgver=25.12.3
+pkgver=26.04.0
 pkgrel=1
 pkgdesc="Computer-Aided Translation System (mingw-w64)"
 arch=('any')
@@ -52,7 +52,7 @@ groups=(
 )
 source=("https://download.kde.org/stable/release-service/${pkgver}/src/${_realname}-${pkgver}.tar.xz"{,.sig}
         '0002-lokalize-22.04.3-fix-cast.patch')
-sha256sums=('f62c5c1116d0fa8c7c3d24b1b642060bc3a0fddc6aa65218f375303de4191e8e'
+sha256sums=('d72c4d817b529c252eff748792676ee8535f5928642701ec4e18b36d066f64f9'
             'SKIP'
             'd11a8f48dfc0dcd8d85f0b364a917f40d5d7b01bcd74726336c557e6e18339fe')
 validpgpkeys=(CA262C6C83DE4D2FB28A332A3A6A4DB839EAA6D7  # Albert Astals Cid <aacid@kde.org>

--- a/mingw-w64-umbrello/PKGBUILD
+++ b/mingw-w64-umbrello/PKGBUILD
@@ -4,8 +4,8 @@ source "$(dirname ${BASH_SOURCE[0]})"/../mingw-w64-PKGBUILD-common/kde-framework
 _realname=umbrello
 pkgbase=mingw-w64-${_realname}
 pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}")
-pkgver=25.12.3
-pkgrel=2
+pkgver=26.04.0
+pkgrel=1
 pkgdesc="UML modeller (mingw-w64)"
 arch=('any')
 mingw_arch=('ucrt64' 'clang64' 'clangarm64')
@@ -41,7 +41,7 @@ makedepends=("${MINGW_PACKAGE_PREFIX}-cc"
              "${MINGW_PACKAGE_PREFIX}-kdoctools"
              "${MINGW_PACKAGE_PREFIX}-icoutils")
 source=("https://download.kde.org/stable/release-service/${pkgver}/src/${_realname}-${pkgver}.tar.xz"{,.sig})
-sha256sums=('45714539df740badcbbd7cba63a8436786ff85ef071af8e5d04f4971e7d62dcb'
+sha256sums=('4685cf3d61cdf2cd6aa1ad524e07e578c60928ff5254fe0ddc5eb2dba7a0cb48'
             'SKIP')
 validpgpkeys=(CA262C6C83DE4D2FB28A332A3A6A4DB839EAA6D7  # Albert Astals Cid <aacid@kde.org>
               F23275E4BF10AFC1DF6914A6DBD2CE893E2D1C87  # Christoph Feck <cfeck@kde.org>

